client/src/styles/Header.css

```diff
@@ -0,0 +1,92 @@
+.header-container {
+  padding: 20px;
+  text-align: center;
+  color: #fff;
+  /* background-color: #191a1d; */
+}
+
+.header-container h1 {
+  color: #fff;
+  font-size: 24px;
+  margin-bottom: 20px;
+}
+
+.video-container iframe {
+  margin: 10px;
+  border-radius: 10px;
+}
+
+.slick-slider {
```

Comment:
Is this component being used? I couldn't see a slider in the header

Reply:
You are absolutely right.
I will go through all of the css and try to look up for unit and sass